Assets/MixedRealityToolkit.Examples/Demos/UX/ManipulationHandler/Scripts/ChangeManipulation.cs

```diff
@@ -31,14 +32,28 @@ public void TryStopManipulation()
         {
             if (manipulatedObject != null && collisionTrigger != null && manipulatedObjCollider != null)
             {
-                if (collisionTrigger.bounds.Intersects(manipulatedObjCollider.bounds))
+                if (!collisionTrigger.bounds.Intersects(manipulatedObjCollider.bounds))
                 {
-                    manipulatedObject.ForceEndManipulation();
+                    return;
+                }
+
+                var mh = manipulatedObject.GetComponent<ManipulationHandler>();
```

Comment:
>mh [](start = 20, length = 2)

Nit: May as well spell out the entire thing (i.e. manipulationHandler and primitiveManipulator).